ivy/functional/frontends/torch/pointwise_ops.py

```diff
@@ -450,3 +450,8 @@ def hypot(input, other, *, out=None):
 @to_ivy_arrays_and_back
 def sigmoid(input, *, out=None):
     return ivy.sigmoid(input, out=out)
+
+
+@to_ivy_arrays_and_back
+def lerp(start, end, weight):
```

Comment:
The first argument should be `input` as stated in the torch docs: https://pytorch.org/docs/stable/generated/torch.lerp.html